examples/patterns/switch.html

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Switch
+category: _patterns
+---
+
+<label id="switch-on-label" for="switch-on">Switch - On</label>
+<button id="switch-on" class="p-switch" type="button" role="switch" aria-checked="true" aria-labelledby="switch-on-label">
+  <span>On</span>
+  <span>Off</span>
+</button>
+
+<label id="switch-off-label" for="switch-off">Switch - Off</label>
+<button id="switch-off" class="p-switch" type="button" role="switch" aria-checked="false" aria-labelledby="switch-off-label">
+  <span>On</span>
+  <span>Off</span>
+</button>
+
+<label id="on-disabled-label" for="on-disabled">Switch - On disabled</label>
+<button id="on-disabled" class="p-switch" type="button" role="switch" aria-checked="true" aria-labelledby="on-disabled-label" disabled>
+  <span>On</span>
+  <span>Off</span>
+</button>
+
+<label id="off-disabled-label" for="off-disabled">Switch - Off disabled</label>
+<button id="off-disabled" class="p-switch" type="button" role="switch" aria-checked="false" aria-labelledby="off-disabled-label" disabled>
+  <span>On</span>
+  <span>Off</span>
+</button>
+
+<h2>Example JavaScript</h2>
+<pre><code>
+(function() {
+  /**
+   * Control switch elements via toggling aria-checked
+   *
+   * @param {String} switchBtn
```

Comment:
We don't currently do anything with JSdoc comments and including it furthers adds to the volume and complexity of example code, can you remove please?